lua/entities/hl_camera_ghost/shared.lua

```diff
@@ -0,0 +1,13 @@
+ENT.Type = "anim"
+ENT.Base = "base_gmodentity"
+
+ENT.RenderGroup = RENDERGROUP_BOTH
```

Comment:
Missed this before - why is it RENDERGROUP_BOTH instead of just RENDERGROUP_TRANSLUCENT?